web/src/settings_components.ts

```diff
@@ -164,7 +164,7 @@ export function get_property_value(
     }
 
     // eslint-disable-next-line @typescript-eslint/consistent-type-assertions
-    return realm[property_name as keyof RealmSetting] as valueof<RealmSetting>;
+    return realm[property_name as keyof RealmSetting];
```

Comment:
After updating the `realm_schema` linter is suggesting to remove this type assertion.

Reply:
Should `// eslint-disable-next-line @typescript-eslint/consistent-type-assertions` also be removed then?

Reply:
We are still using the `as` operator to assert property_name to be the key of RealmSettings.

Reply:
Ahh, quite so.